scout/reports.md

```diff
@@ -0,0 +1,97 @@
+---
+description: More details on the Docker Scout Advisory Database and CVE-to-package matching service.
+keywords: scanning, analysis, vulnerabilities, Hub, supply chain, security, report, reports
+title: Analysis reports
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout reports help you share a report of the analysis of images in a namespace with your team. The reports helps team members in roles such as security, compliance, and operations to know what vulnerabilities and issues they need to focus on.
```

Comment:
`the reports help`?

Reply:
I think we should mention developers here, this isn't purely a security/compliance play here. Something like 
> Developers can now see an overview of their security status across all their images from both Docker Hub and Artifactory, and get remediation advice at their fingertips.

Reply:
Haha, I was given the impression it was, but will add them in :)